WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -130,6 +135,20 @@ class NotificationsViewController : UITableViewController
         tableViewHandler.updateRowAnimation = .None
     }
 
+    override func viewDidDisappear(animated: Bool) {
+        super.viewDidDisappear(animated)
+
+        // Glitch Workaround. Scenario:
+        //  -   Mark as Read saves the mainMOC
+        //  -   Simperium sends the change
+        //  -   The backend acknowledges the change, and merges back into the mainMOC
+        // This causes a reloadData event, that might cut any ongoing animations in the detail views. For that reason,
```

Comment:
Just for the record: discussed at lenght over slack, HUGE thanks for your time Diego!!!